Core/GDCore/Project/EventsFunctionsExtension.cpp

```diff
@@ -86,6 +86,12 @@ void EventsFunctionsExtension::SerializeTo(SerializerElement& element) const {
 }
 
 void EventsFunctionsExtension::UnserializeFrom(
+    gd::Project& project, const SerializerElement& element) {
```

Comment:
Nice, I think a UnserializeFrom should indeed do the entire job - always.
If you want to partially unserialize, then you use the other functions.